examples/Makefile

```diff
@@ -60,7 +60,7 @@ package-examples: package-example-big-bang package-example-software-factory pack
 
 .PHONY: package-example-big-bang
 package-example-big-bang: ## Create the Big Bang Core example
-	cd big-bang && $(ZARF_BIN) package create --confirm && mv zarf-package-* ../sync/
+	cd big-bang && $(ZARF_BIN) package create -l debug --confirm && mv zarf-package-* ../sync/
```

Comment:
Was this debug flag left intentionally?

Reply:
No, but now that it is there I guess I kinda like it being there. Gives a lot more context as the thing is being built. 

But I can take it out if you think that would be best

Reply:
Debug exposed to users is odd to me, but the fact that we still are even using make files exposed for normal users is totally wrong, so I think we can leave it until we finish getting rid of make files for non-dev work.